[2025-08-05T19:04:34.350+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-05T19:04:18.558608+00:00 [queued]>
[2025-08-05T19:04:34.385+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-05T19:04:18.558608+00:00 [queued]>
[2025-08-05T19:04:34.386+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-05T19:04:34.434+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): crea_vista_analitica> on 2025-08-05 19:04:18.558608+00:00
[2025-08-05T19:04:34.440+0000] {standard_task_runner.py:60} INFO - Started process 79 to run task
[2025-08-05T19:04:34.446+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'crea_vista_analitica', 'manual__2025-08-05T19:04:18.558608+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmplzh1zwb2']
[2025-08-05T19:04:34.449+0000] {standard_task_runner.py:88} INFO - Job 228: Subtask crea_vista_analitica
[2025-08-05T19:04:34.565+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-05T19:04:18.558608+00:00 [running]> on host 70eaa744a05c
[2025-08-05T19:04:34.747+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='crea_vista_analitica' AIRFLOW_CTX_EXECUTION_DATE='2025-08-05T19:04:18.558608+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-05T19:04:18.558608+00:00'
[2025-08-05T19:04:34.748+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-05T19:04:34.749+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/postgres_04_crea_vista_analitica.py']
[2025-08-05T19:04:34.757+0000] {subprocess.py:86} INFO - Output:
[2025-08-05T19:04:34.954+0000] {subprocess.py:93} INFO - Conectado a PostgreSQL en postgres:5432/medallon_db
[2025-08-05T19:04:34.964+0000] {subprocess.py:93} INFO - Ejecutando: /opt/airflow/sql/04_vista_analisis.sql
[2025-08-05T19:04:34.965+0000] {subprocess.py:93} INFO - Vista de an√°lisis creada/reemplazada correctamente
[2025-08-05T19:04:34.966+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-05T19:04:35.021+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=crea_vista_analitica, execution_date=20250805T190418, start_date=20250805T190434, end_date=20250805T190435
[2025-08-05T19:04:35.101+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-05T19:04:35.163+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
